src/pageContent/Aksara/index.tsx

```diff
@@ -0,0 +1,189 @@
+import React from "react";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Button from "src/components/Button";
+import Text from "src/components/Text";
+import AksaraBox from "src/components/AksaraBox";
+import * as S from "./styles";
+import { textArray } from "./stubs";
+
+const AksaraContent: React.FC = () => {
+  const router = useRouter();
+
+  const titles = ["Manusia", "Opini", "Modulus"];
+  const headerContent = {
+    Manusia:
+      "A collection of stories from Indonesian students in NUS of how life in Singapore can transform the ideas of identity, community, and nationality among Indonesians.",
+    Opini:
+      "A constructive platform where one’s interests, concerns, and perceptions of the ever-evolving world could be conveyed and shared freely through writings.",
+    Modulus:
+      "A module review platform which allows fellow Indonesians to share and obtain more personal and candid module reviews.",
+  };
+
+  const renderHeader = (title) => {
+    return (
+      <div
+        className={`bg-aksaraBox bg-no-repeat flex flex-col justify-center lg-min:p-5 mt-4 break-words lg-min:h-96  lg-min:max-w-lg xl-min:max-w-lg lg-min:w-72 xl-min:w-full`}
+      >
+        <div className={`m-2 text-xl lg-min:text-4xl`}>
+          <Text variant="aksaraTextheader">{title}</Text>
+        </div>
+        <div className={`m-2 max-w-lg xl-min:w-96`}>
+          <div className={`w-9/10`}>{headerContent[title]}</div>
+        </div>
+      </div>
+    );
+  };
+
+  const renderSection = (title) => {
+    return (
+      <div className={`flex flex-col items-center mt-10`}>
+        <div>
+          <div
+            className={`border-b-2 border-black font-mono text-2xl lg-min:text-4xl`}
+          >
+            <Text variant="aksaraTextheader">{title}</Text>
+          </div>
+        </div>
+        <div
+          className={`flex flex-col justify-evenly flex-wrap lg-min:flex-row lg-min:justify-between lg-min:max-w-screen-lg`}
+        >
+          {textArray.map((item) => {
+            return (
+              <div className={`mt-10 lg-min:m-8`}>
+                <AksaraBox
+                  title={item.title}
+                  description={item.description}
+                  hyperlink="https://aksarapinus.wordpress.com/"
+                ></AksaraBox>
+              </div>
+            );
+          })}
+        </div>
+        <div className={`mt-2 lg-min:mt-0 lg-min:mb-10`}>
+          <Button variant="secondary" onClick={() => router.push("/about")}>
```

Comment:
Can make it such that when you click it actually loads more.

Limit the render to just 3 boxes first until user clicks `Load More`